src/client/android/app/src/main/java/com/yoda/MainActivity.java

```diff
@@ -12,4 +15,10 @@
     protected String getMainComponentName() {
         return "yoda";
```

Comment:
What's the purpose of this method? There's gotta be a better way to construct this string. Perhaps declare it as constant at the top and use it here?